lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -541,13 +574,52 @@ def __get_template_with_version(self):
 
         return template
 
+    def __get_storage_domain_and_all_template_disks(self, template):
+
+        if self._module.params.get('template') is None:
```

Comment:
`if self._module.params.get('template') is None:`
->
`if self.param('template') is None:`